setup.cfg

```diff
@@ -1,4 +1,8 @@
 [flake8]
+# Ignore undefined directives. These are handled by sphinx, and flake8 doesn't
+# know about them.
+# Also ignore "W504 line break {after,before} binary operator"
+ignore = W503,W504,RST303
```

Comment:
this is the culprit! please use `extend-ignore` instead of `ignore`, and roll back changes in files that you don't touch.

Reply:
(i'd rather fix style issues later... or more gradually)